subjects/lem-in/audit/README.md

```diff
@@ -293,11 +293,11 @@ $
 
 ###### Does it present at least the result above?
 
-##### Try running the program with example06.txt and with 100 ants.
+##### Try running the program with [example06](https://public.01-edu.org/subjects/lem-in/examples/) and with 100 ants.
```

Comment:
Please avoid GitHub links, as they force us to rely on it and it sometimes refuse to serve subjects (rate-limitation).

Use relative links instead or absolute ones but to ours domains, not github.com